### ISSUE TITLE
Any pretrained agents for the MuJoCo envs for OpenAI/baselines?

### ISSUE THREAD

**Issue by benelot** [openai/baselines#384]
Hello,

I think I would be very helpful to have agents that are pre-trained on different gym environments. I am working on some transfer learning examples and it could be very helpful to have some baselines. Does anyone have pre-trained agents to experiment with?

**Comment by ling-pan** [comment]
Agree. I am also looking for the implementations of a2c algorithm in MuJoCo envs. It would be very helpful if someone can share the pre-trained models.

**Comment by jeremyf21** [comment]
It also seems like the baselines algorithms are not compatible with action spaces that are not defined by integers (i.e. continuous space in the FetchReach robot)

**Comment by pzhokhov** [comment]
As a part of the code quality improvement effort, it was decided to include the hyperparameters and document all tips and tricks needed to reproduce state-of-the-art results, but not include the trained models themselves, at least not in the first pass before we figure out good APIs (because as it stands at the moment different policy / model apis make transfer learning experiments difficult).
